src/test/java/pages/HomePage.java

```diff
@@ -0,0 +1,109 @@
+package pages;
+
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.time.Duration;
+
+public class HomePage extends BasePage {
+
+    WebDriver driver;
+    WebDriverWait wait;
+
+    //WebElements
+    @FindBy(css = "img.avatar")
+    WebElement userAvatarIcon;
+//    By userAvatarIcon = By.cssSelector("img.avatar");
+
+    @FindBy (css = "i.fa.fa-plus-circle")
+    WebElement addPlaylistButton;
+//    By addPlaylistButton = By.cssSelector("i.fa.fa-plus-circle");
+
+    @FindBy (xpath = "//li[text()='New Playlist']")
+    WebElement addNewPlaylistButton;
+//    By addNewPlaylistButton = By.xpath("//li[text()='New Playlist']");
```

Comment:
I recommend not using code comments: for, example, when you have a code that cannot be understood, then comments are not the solution. Try to rewrite the code or rename fields and other elements in a way that the reader will understand the action that you are doing there.

Reply:
Thank you so much for the suggestions! It helped out a lot. I edited my code to reflect the changes. Thanks again! :)